sympy/printing/str.py

```diff
@@ -402,7 +402,8 @@ def _print_FracElement(self, frac):
             return numer + "/" + denom
 
     def _print_Poly(self, expr):
-        terms, gens = [], [ self._print(s) for s in expr.gens ]
+        ATOM_PREC = PRECEDENCE["Atom"] - 1
```

Comment:
@asmeurer if we use 
```
ATOM_PREC = PRECEDENCE["Atom"]
```
Output::
```
>>> str(Poly(0, x))
"Poly(0, (x), domain='ZZ')"
```
in that case we don't need ```(x)``` we only need ```x```.

if we use 
```
ATOM_PREC = PRECEDENCE["Atom"] - 1
```
Output::
```
>>> str(Poly(0, x))
"Poly(0, x, domain='ZZ')"
```